.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,21 +31,40 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] Visual Studio
 - [ ] Eclipse IDE
 - [ ] VSCode
+- [ ] When compiling the compliler flag was used ""enable /Wall -Wformat-security -Werror=format-security"
+
+**_Delete this line and everything below if this is not a PR for a new sample_**
+
+**_Delete this line and all above it if this PR is for a new Sample_**
+# Adding a New samples
+##Introduction
 
-## Checklist for New samples:
+The full instructions on submitting a new sample is located in the [wiki](https://github.com/oneapi-src/oneAPI-samples/wiki/Contributing-a-New-Sample) if you have any questions about the steps for submitting a new code sample
 
+## Checklist
+Administrative
 - [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
+- [ ] Review Sample with Domain Expert: <insert NameHere>
+- [ ] If you dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster
+
+Code Development
 - [ ] Implement coding guidelines and ensure code quality.
-- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
-- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
 - [ ] Adhere to readme template 
-- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
 - [ ] Enforce format via clang-format config file
+- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
+- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
+- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
```

Comment:
Is this not part of the CI?  If so, I don't think we need this in the checklist.

Reply:
yes and no... 
The first  two are just referring to proper json construction
1. Sample browser setup
2. actually running the ci tests
3. the jsonint is preventative to keep the ci from failing